examples/complete-mssql/main.tf

```diff
@@ -157,3 +157,51 @@ module "db" {
 
   tags = local.tags
 }
+
+module "db_random_password" {
+  source = "../../"
+
+  identifier = local.name
+
+  engine               = "sqlserver-ex"
+  engine_version       = "15.00.4073.23.v1"
+  family               = "sqlserver-ex-15.0" # DB parameter group
+  major_engine_version = "15.00"             # DB option group
+  instance_class       = "db.t3.large"
+
+  allocated_storage     = 20
+  max_allocated_storage = 100
+  storage_encrypted     = false
+
+  name                   = null
+  username               = "complete_mssql"
+  create_random_password = true
```

Comment:
we can just move lines 178-179 up to replace line 130; we don't want to add too many resources since it takes quite a long time already to spin up to test and plus the identifiers will conflict